samples/cellular/modem_shell/src/link/link_shell.c

```diff
@@ -2597,6 +2602,10 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 		funmode, NULL,
 		"Set/read functional modes of the modem.",
 		link_shell_funmode, 0, 10),
+	SHELL_CMD_ARG(
+		ifaddrs, NULL,
+		"Get interface address information.",
+		link_shell_ifaddrs, 0, 0),
```

Comment:
To return an error in case any arguments are given:
```suggestion
		link_shell_ifaddrs, 1, 0),
```

Reply:
Fixed.